src/BloomExe/Publish/Epub/PublishEpubApi.cs

```diff
@@ -161,6 +152,35 @@ public void RegisterWithApiHandler(BloomApiHandler apiHandler)
 		}
 
 		private void HandleEpubSave(ApiRequest request)
+		{
```

Comment:
Should we have a `Debug.Assert(ControlForInvoke != null || Program.RunningUnitTests || Program.RunningOnUiThread);` here? If so, what is `Debug` and where does it come from?